games-fps/lab3d/lab3d-3.0.1.recipe

```diff
@@ -0,0 +1,88 @@
+SUMMARY="A port of Ken's Labyrinth to modern systems"
+DESCRIPTION="LAB3D is a port of Ken's Labyrinth to modern operating systems, using OpenGL \
+for 3D accelerated graphics output and the SDL library to provide user input, sound output, \
+threading, unaccelerated graphics and some graphics support functions. Music output is through \
+Adlib emulation (recommended) or MIDI (MIDI only on Windows, Linux and other operating systems \
+with OSS-compatible sound APIs).
+
+As this program is provided for free, neither Ken Silverman nor Jan Lönnberg take any \
+responsibility for any effects, adverse or otherwise, resulting from the use of this program.
+
+Current version: 3.0.1 (16/12/2013).
+
+Improvements over the original Ken's Labyrinth:
+
+- Runs natively on 32-bit and 64-bit Windows or Linux (and possibly some similar systems).
+- Uses OpenGL to provide hardware accelerated, anti-aliased graphics with trilinear interpolation in true colour (where available).
+- Multiple simultaneous sound effects in stereo.
+- Many bug fixes."
+HOMEPAGE="http://icculus.org/LAB3D/"
+COPYRIGHT="1992-1993 Ken Silverman
+	2002-2014 Jan Lönnberg"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://icculus.org/LAB3D/LAB3D-SDL-$portVersion.tar.gz"
+CHECKSUM_SHA256="67b4bf5c5691e83ea7ac20221f8c03f6fb9a205ff6ef77f3cb70ecfa32c35738"
+SOURCE_URI_2="http://advsys.net/ken/klab/labfull.zip"
+CHECKSUM_SHA256_2="643aa0ca8d1a7d2c88afecd952da00f3b912be8b83c4c1d2f4ab9b50491fadfd"
+SOURCE_DIR="LAB3D-SDL-$portVersion"
+ADDITIONAL_FILES="lab3d-$portVersion.rdef.in"
+PATCHES="lab3d-$portVersion.patchset"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	lab3d$secondaryArchSuffix = $portVersion
+	app:KensLabyrinth
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libGLU$secondaryArchSuffix
+	lib:libSDL_1.2$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libGL$secondaryArchSuffix
+	devel:libGLU$secondaryArchSuffix
+	devel:libSDL_1.2$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	"
+
+BUILD()
+{
+	make $jobArgs -f Makefile.Unix
+}
+
+INSTALL()
+{
+	mkdir -p $appsDir/KensLabyrinth
+	cp -a $sourceDir/ken $appsDir/KensLabyrinth/KensLabyrinth
+
+	local APP_SIGNATURE="application/x-vnd.KensLabyrinth"
+	local MAJOR="`echo "$portVersion" | cut -d. -f1`"
+	local MIDDLE="`echo "$portVersion" | cut -d. -f2`"
+	local MINOR="`echo "$portVersion" | cut -d. -f3`"
+	local LONG_INFO="$SUMMARY"
+	sed \
+		-e "s|@APP_SIGNATURE@|$APP_SIGNATURE|" \
+		-e "s|@MAJOR@|$MAJOR|" \
+		-e "s|@MIDDLE@|$MIDDLE|" \
+		-e "s|@MINOR@|$MINOR|" \
+		-e "s|@LONG_INFO@|$LONG_INFO|" \
+		$portDir/additional-files/lab3d-$portVersion.rdef.in > lab3d-$portVersion.rdef
+
+	addResourcesToBinaries lab3d-$portVersion.rdef $appsDir/KensLabyrinth/KensLabyrinth
+	mimeset -f $appsDir/KensLabyrinth/KensLabyrinth
+
+	cp -a $sourceDir2/* $appsDir/KensLabyrinth/
```

Comment:
No need to copy everything over (don't need the .exe for instance), this should be sufficient:
```diff
-	cp -a $sourceDir2/* $appsDir/KensLabyrinth/
+	cp -a $sourceDir2/{*.DAT,*.KZP} $appsDir/KensLabyrinth/
```